dissect/target/helpers/nfs/nfs3.py

```diff
@@ -167,3 +164,22 @@ class Read3resok:
     count: int
     eof: bool
     data: bytes
+
+
+@dataclass
+class DirOpArgs3:
+    handle: FileHandle3
+    filename: str
+
+
+@dataclass
+class Lookup3resok:
```

Comment:
Should anything in `resok` not be capitalized? I'm not entirely sure what `resok` means here.